tests/tasks/setup/kit-cluster/default.yaml

```diff
@@ -17,10 +17,27 @@ spec:
   - name: guest-cluster-region
     default: "us-west-2"
     description: The region where the kit cluster is created.
-  - name: control-plane-spec
-    description:  url to the controlplane spec.
-  - name: dataplane-spec
-    description: url to the dataplane spec.
+  - name: version
+    default: "1.21"
```

Comment:
Not clear to me why do we need defaults in two places, pipelines and here?

Reply:
Do we want that, as its a lot of duplication and if we need to change default we have to change them at 2 locations. Not a blocker for now but we should reconsider this.